doc/ref_model/chapters/chapter04.rst

```diff
@@ -0,0 +1,500 @@
+Profile Extension        pe       ❌                           ✅                                      Specifies the `profile extensions <#profile-extensions>`__.                                                                                                                                                                         Optional
+Profile Extra Specs      pes      ❌                           ✅                                      Specifies special node configurations not accounted for by the profile and profile extensions.                                                                                                                                      Optional
+======================== ======== =========================== ====================================== ===================================================================================================================================================================================================================================== ==========================
+
+**Table 4-13:** Resource Flavours (complete list of Workload Capabilities) Specifications
+
+Virtual Network Interface Specifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The virtual network interface specifications extend a Flavour customisation with network interface(s), with an associated bandwidth, and are identified by the literal, “n”, followed by the interface bandwidth (in Gbps). Multiple network interfaces can be specified by repeating the “n” option.
+
+Virtual interfaces may be of an Access type, and thereby untagged, or may be of a Trunk type, with one or more 802.1Q tagged logical interfaces. Note, tagged interfaces are encapsulated by the Overlay, such that tenant isolation (i.e., security) is maintained, irrespective of the tag value(s) applied by the workload.
+
+Note, the number of virtual network interfaces, aka vNICs, associated with a virtual compute instance, is directly related to the number of vNIC extensions declared for the environment. The vNIC extension is not part of the base Flavour.
+
+::
+
+   <network interface bandwidth option> :: <”n”><number (bandwidth in Gbps)>
+
+================================== =================================
+Virtual Network Interface Option   Interface Bandwidth
+================================== =================================
+n1, n2, n3, n4, n5, n6             1, 2, 3, 4, 5, 6 Gbps
+n10, n20, n30, n40, n50, n60       10, 20, 30, 40, 50, 60 Gbps
+n25, n50, n75, n100, n125, n150    25, 50, 75, 100, 125, 150 Gbps
+n50, n100, n150, n200, n250, n300  50, 100, 150, 200, 250, 300 Gbps
+n100, n200, n300, n400, n500, n600 100, 200, 300, 400, 500, 600 Gbps
+================================== =================================
+
+**Table 4-14:** Virtual Network Interface Specification Examples
+
+Storage Extensions
```

Comment:
we need to define a label as this section is referenced externally
```suggestion
.. _RM-Ch04-Storage-Extensions:
Storage Extensions
```

Reply:
I believe all headers got a label automatically and can be referenced externally.

Reply:
It actually depends on if we decide to use the [sphinx.ext.autosectionlabel](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html) extension. Related discussion is in https://github.com/cntt-n/CNTT/pull/2737#discussion_r776459859